portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,33 @@
 
 package com.google.sps.servlets;
 
+import com.google.sps.storage.CommentDatastore;
+import com.google.sps.util.JsonUtil;
+
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private CommentDatastore commentDatastore;
```

Comment:
since it is a datastore, we want it to be shared across all instance of the "frontend-backend" (dataservlet)
to make it shared across all instance, lets make it a static